08-transport.Rmd

```diff
@@ -117,24 +125,24 @@ These zone-level data are small but often vital for gaining a basic understandin
+Although transport systems are primarily based on linear features and nodes --- including pathways and stations --- it often makes sense to start with areal data <!-- source and/or example -->.
+Three zone types will typically be of particular interest: the study region, origin zones (typically residential areas), and destination zones (which are often the same as the origin zones). <!-- this implies people are going from their origin to the origin possibly via a detour (which might be work/supermarket/etc.--> 
 
 The simplest way to define a study area is often the first matching boundary returned by OpenStreetMap, which can be obtained using **osmdata** as follows:
 
 ```{r, eval=FALSE}
 region_bb = osmdata::getbb("Bristol", format_out = "sf_polygon")
 ```
 
-The result is an `sf` object representing the bounds of the largest matching city region, either a rectangular polygon of the bounding box or detailed polygonal boundary.^[
-In cases where the first match does not provide the right name, the country or region should be specified, for example `Bristol Tennessee` for an America Bristol.
+The result is an `sf` object representing the bounds of the largest matching city region, either a rectangular polygon of the bounding box or a detailed polygonal boundary.^[
+In cases where the first match does not provide the right name, the country or region should be specified, for example `Bristol Tennessee` for a Bristol located in America.
 ]
-In this case the command returns a detailed polygon representing official boundary of Bristol (see the inner blue boundary in Figure \@ref(fig:bristol)).
-This approach is not always appropriate, because boundaries in OSM often differ from boundaries used by local authorities.
-Furthermore, even official boundaries returned by OSM may not be appropriate for transport research because they bear little relation to where people travel.
+In this case the command returns a detailed polygon representing the official boundary of Bristol (see the inner blue boundary in Figure \@ref(fig:bristol)).
+A word of warning: First, OSM boundaries as returned by OSM are not necessarily the official boundaries provided by local authorities.
+Secondly, even if OSM returns the official boundaries, these still may not be appropriate for transport research because they bear little relation to where people travel.
 
-Travel to Work Areas (TTWAs) overcome this issue by creating a zoning system analogous to hydrological watersheds.
-TTWAs were first defined as contiguous zones within which 75% of the population travels work [@coombes_efficient_1986], and this is the definition used in this chapter.
+Travel to Work Areas (TTWAs) overcome the second issue by creating a zoning system analogous to hydrological watersheds.
+TTWAs were first defined as contiguous zones within which 75% of the population travels to work [@coombes_efficient_1986], and this is the definition used in this chapter.
```

Comment:
Pls check if `travels to work` is what you mean!

Reply:
Yes - areas in which people who live there also work there.